src/inference/include/openvino/runtime/core.hpp

```diff
@@ -567,22 +578,14 @@ class OPENVINO_RUNTIME_API Core {
      * operation.
      * @return A compiled model.
      */
-    CompiledModel import_model(std::istream& model_stream, const RemoteContext& context, const AnyMap& properties = {});
+    CompiledModel import_model(ov::Tensor& model, const RemoteContext& context, const AnyMap& properties = {});
 
-    /**
-     * @brief Imports a compiled model from the previously exported one with the specified remote context.
-     * @tparam Properties Should be the pack of `std::pair<std::string, ov::Any>` types.
-     * @param model_stream Model stream.
-     * @param context Pointer to a RemoteContext object.
-     * @param properties Optional pack of pairs: (property name, property value) relevant only for this
-     * load operation.
-     * @return A compiled model.
-     */
-    template <typename... Properties>
-    util::EnableIfAllStringAny<CompiledModel, Properties...> import_model(std::istream& model_stream,
-                                                                          const RemoteContext& context,
+
+    template <typename MODEL_TYPE, typename DEVICE_OR_CONTEXT_T, typename... Properties>
+    util::EnableIfAllStringAny<CompiledModel, Properties...> import_model(MODEL_TYPE& model,
```

Comment:
Lets keep other version just leave properties as template parameters
The description will be more accurate fo context or device also model type can be stream or blob